### PR TITLE
Update uninstall.sh and install.sh

### PR DESCRIPTION
**Pull request by AnujK2901**
Added more comments for new contributors.

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -6,53 +6,65 @@ GREEN='\033[0;32m'
...
     echo ""
     exit 1
   fi
 fi
 
-# Check up and clean old verisons
+# Check up and clean old versions
 echo "Checking and cleaning up old $1 files"
 
+# Removing ZIP File
```
Comment (manishprivet) [inline install.sh]:
```suggestion
# Removing ZIP File if exists
```

File: install.sh [hunk 2]
```diff
@@ -6,53 +6,65 @@ GREEN='\033[0;32m'
...
 
+# Adding Wallpapers starts from here
 echo "Adding $1 to wallpapers ..."
 
 # Create directories if they doesn't exist
 mkdir -p $LOCATION/backgrounds/gnome
 mkdir -p $LOCATION/gnome-background-properties
 
-# Download files from the zip repo
+# Download files from the zip repository
```
Comment (manishprivet) [inline install.sh]: It's actually not a repository, but an oracle cloud object storage bucket

**Review by manishprivet — COMMENTED** [review]
Thank you so much for your contribution . It really does increase the readability of the script 😄

Left some comments though for minor changes

**Comment by AnujK2901** [comment]
All minor changes are added to my commit.

**Comment by AnujK2901** [comment]
All minor changes are added to my commit.

**Comment by AnujK2901** [comment]
> All minor changes are added to my commit.

3rd commit has all the necessary changes.

**Comment by manishprivet** [comment]
Thanks @AnujK2901 🚀🚀

**Review by manishprivet — APPROVED** [review]

**Comment by AnujK2901** [comment]
> Thanks @AnujK2901 🚀🚀

Kindly Merge these changes by selecting "Squash and merge" :)

**Pull request merged**